doc/COMPILING/COMPILING.md

```diff
@@ -632,7 +632,7 @@ There are reports of CDDA building fine on recent OpenBSD and FreeBSD machines (
 
 ### Building on FreeBSD/amd64 13.0 with the system compiler
 
-FreeBSD uses clang as the default compiler as of 10.0, and combines it with libc++ to provide C++14 support out of the box.
+FreeBSD provides C++17 support out of the box since 12.0, and uses clang as the default compiler.
```

Comment:
The wording is a bit strange, it's the compiler and standard library together provides support for a particular language standard, not the operating system.